<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dcard</class>
 <widget class="QDialog" name="dcard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UID</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>671</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledelete">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
